Content.Server/Alert/Click/BreakVow.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.Alert;
+using Content.Server.Abilities.Mime;
+
+namespace Content.Server.Alert.Click
+{
+    ///<summary>
+    /// Break your mime vows
+    ///</summary>
+    [DataDefinition]
+    public sealed class BreakVow : IAlertClick
+    {
+        public void AlertClicked(EntityUid player)
+        {
+           if (IoCManager.Resolve<IEntityManager>().TryGetComponent<MimePowersComponent?>(player, out var mimePowers))
+           {
+                EntitySystem.Get<MimePowersSystem>().BreakVow(player, mimePowers);
+           }
+        }
+    }
```

Comment:
breaking/retaking the vow should really be actions IMO. also can make them yaml prototype actions

Reply:
actuaaaaally im not sure. having it be an alert click is fine, i think? can get a second opinion on that

Reply:
Chose alert clicks because those are harder to do accidentally

Reply:
also true